OpenEdXMobile/res/layout/activity_single_fragment_base.xml

```diff
@@ -6,65 +6,70 @@
     android:background="@color/grey_act_background"
     android:orientation="vertical"
     android:splitMotionEvents="false"
-    android:fitsSystemWindows="true"
-    >
+    android:fitsSystemWindows="true">
```

Comment:
Is seems that this is needed for the drawer to draw the status bar background, and inset it's contents to prevent overlay with it, since we're using the translucent status bar from the Material design in Lollipop+.